css-two-column-layout/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <section>
+    <h1>SCALE, SPEED, AGILITY</h1>
+    <label>We've dedicated thousands of man hours to making healthcare institutions in the Helium<br>
+      network run 10x more efficiently and deliver an unparalled quality of care.</label>
+  </section>
+  <div class="container">
+    <div class="row">
+      <div class="column-two-third">
+        <img src="images/software.svg" alt="Software">
+      </div>
+      <div class="column-two-third">
+        <h2><b>INTUITIVE,<br>SMART SOFTWARE</b></h2>
+        <p>We build products that transform how<br>
+          healthcare facilities run across Africa<br>&#8212 making them radically more efficient<br>
+          operationally and financially, a complete<br>
+          turnaround.</p>
+      </div>
+    </div>
+    <div class="column-two-third">
+      <img src="images/practice.svg" alt="practice">
+    </div>
+    <div class="column-two-third">
+      <h3><b>PRACTICE MEDICINE, NOT<br>
+          ADMINISTRATION</b></h3>
+      <p>Our all in-one electronic health records<br>
+        (EHR), hospital management (HMIS) &<br>
+        billing system helps streamline your<br>
+        facility, reduce waste and improve<br>quality of care.</p>
+    </div>
+    <div class="row">
+      <div class="column-two-third">
+        <img src="images/circumstance.svg" alt="circumstance">
+      </div>
+      <div class="column-two-third">
+        <h4><b>FOR ANY CIRCUMSTANCE</b></h4>
+        <p>Our intuitive system is built around the <br>
+          peculiarities of your facility; we support<br>
+          offline deployments, sync to the cloud<br>
+          and run on any device 24/7.</p>
+      </div>
+    </div>
```

Comment:
each of these groups should be wrapped in a `column-half` class so that the content is grouped into two columns per row rather than the current four.